heidegger_index/constants.py

```diff
@@ -1,6 +1,11 @@
 PERSON = "p"
 WORK = "w"
-LEMMA_TYPES = {PERSON: "person", WORK: "work"}
+GEOGRAPHICAL_NAME = "g"
+LEMMA_TYPES = {PERSON: "person", WORK: "work", GEOGRAPHICAL_NAME: "geographical place name"}
+
+URN = "cts_urn"
+GND = "gnd_id"
+METADATA_TYPES = {URN: "cts_urn", GND: "gnd_id"}
```

Comment:
Misschien kan je dit wat inkorten: `cts` (of `urn`) en `gnd`. Anders moet je zoveel typen elke keer.